api/identifiers/serializers.py

```diff
@@ -65,8 +64,7 @@ def create(self, validated_data):
                 identifiers = get_or_create_identifiers(node)
             except HTTPError:
                 raise exceptions.ValidationError('Error response from client.')
-            except TypeError:
-                raise ServiceUnavailableError()
+
```

Comment:
Hesitant to remove error captures unless we know that either A. The service won't throw that error anymore, or B. That error won't affect process to finish.

Reply:
Yeah, worth noting if their really isn't a good connection, this doesn't get triggered.